### PR TITLE
Fix group creation for in place group selector.

### PR DESCRIPTION
**Pull request by rmjohnson**

### CODE REVIEW

File: components/group-selector/large/component.jsx [hunk 1]
```diff
@@ -37,20 +37,23 @@ export class LargeGroupSelector extends React.Component {
...
-	searchResultsRef = React.createRef();
-	fixedCreateWrapper = false;
+		this.state = {
+			searchInputValue: '',
+			newChurchName: '',
+			newChurchLocation: '',
+			modalContent: 'main',
+			selectedGroupId: -1,
+			createGroupFixed: false,
+			resultsTopMargin: props.showInPlace ? 0 : defaultResultsTopMargin,
```
Comment (dustinsoftware) [inline components/group-selector/large/component.jsx]: I'm pretty sure `this.props` can be referenced using class field syntax.

File: components/group-selector/large/component.jsx [hunk 2]
```diff
@@ -173,6 +176,8 @@ export class LargeGroupSelector extends React.Component {
 	};
 
 	handleScroll = scrollData => {
+		if (this.props.showInPlace) return;
```
Comment (dustinsoftware) [inline components/group-selector/large/component.jsx]: Note to self: Add a lint rule requiring curly braces.

File: components/group-selector/large/component.jsx (hunk 1 above)
Comment (rmjohnson) [reply components/group-selector/large/component.jsx]: i.e. I don't need to put this in a `constructor`? I'll try that. I always thought you had to do a constructor if initial state was based off of props.

File: components/group-selector/large/component.jsx [hunk 3]
```diff
@@ -44,11 +44,10 @@ export class LargeGroupSelector extends React.Component {
 		modalContent: 'main',
 		selectedGroupId: -1,
 		createGroupFixed: false,
-		resultsTopMargin: defaultResultsTopMargin,
+		resultsTopMargin: this.props.showInPlace ? 0 : defaultResultsTopMargin,
 		scrollWidthDelta: 0,
 	};
 
-	modalRef = React.createRef();
```
Comment (rmjohnson) [inline components/group-selector/large/component.jsx]: This *appeared* to be unused.

**Comment by rmjohnson** [comment]
@dustinsoftware, updated!

**Pull request merged**